@here/harp-mapview/lib/poi/PoiManager.ts

```diff
@@ -596,21 +596,31 @@ export class PoiManager {
                 featureId,
                 iconBrightness: technique.iconBrightness,
                 iconColor,
-                iconMinZoomLevel: technique.iconMinZoomLevel,
-                iconMaxZoomLevel: technique.iconMaxZoomLevel,
-                textMinZoomLevel: technique.textMinZoomLevel,
-                textMaxZoomLevel: technique.textMaxZoomLevel
+                iconMinZoomLevel:
+                    getPropertyValue(technique.iconMinZoomLevel ?? technique.minZoomLevel, env) ??
+                    undefined,
+                iconMaxZoomLevel:
+                    getPropertyValue(technique.iconMaxZoomLevel ?? technique.maxZoomLevel, env) ??
+                    undefined,
+                textMinZoomLevel:
+                    getPropertyValue(technique.textMinZoomLevel ?? technique.minZoomLevel, env) ??
+                    undefined,
+                textMaxZoomLevel:
+                    getPropertyValue(technique.textMaxZoomLevel ?? technique.maxZoomLevel, env) ??
+                    undefined
```

Comment:
text limit levels are never read.
To make them work, you can change the check done here:
https://github.com/heremaps/harp.gl/blob/40a856bb3673217fe6cbecbce7f266917164bf60/%40here/harp-mapview/lib/text/TextElementsRenderer.ts#L262

from this:

```
const visibleInZoomLevel =
        poiInfo === undefined ||
        viewState.zoomLevel === undefined ||
        MathUtils.isClamped(
            viewState.zoomLevel,
            poiInfo.iconMinZoomLevel,
            poiInfo.iconMaxZoomLevel
        );
```
to this:

```
const visibleInZoomLevel =
        MathUtils.isClamped(
            viewState.zoomLevel,
            poiInfo.textMinZoomLevel,
            poiInfo.textMaxZoomLevel
        );
```

Reply:
`poiInfo === undefined ` is needed otherwise ` poiInfo.textMinZoomLevel` may be undefined.